### ISSUE TITLE
Lowfi - Add profile updated

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#702]
Mock Link: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/77b5658b-cd35-4eda-b971-f1f4fd0181de/PC-MP-Add-Profile-Empty-7

The current implementation in develop branch for `Add Profile` is different as compared to the new mocks.

Please update this. Also, write test cases for the same.

**Comment by rt4914** [comment]
Assigned to @aggarwalpulkit596

**Comment by aggarwalpulkit596** [comment]
@rt4914 where to get assets such as icons, placeholder images ?

**Comment by rt4914** [comment]
> @rt4914 where to get assets such as icons, placeholder images ?

You can use icons from android studio itself:

`app` Right Click -> New -> Vector Asset and select the icon that you want.

**Comment by aggarwalpulkit596** [comment]
So it does not need be exactly same the adobe files ?🤔

**Comment by rt4914** [comment]
> So it does not need be exactly same the adobe files ?🤔

For now its fine. The issue names contains `Low-fi` which means stands low-fidelity, which means the it should look like mock but if some things are not matching like color/icon/size/etc.

Once this is finished, we normally do `high-fi` which stands for `High- Fidelity` which means that it should match pixel perfect with mock.

**Comment by aggarwalpulkit596** [comment]
@rt4914 Ok thanks for the clarification 👍

**Comment by aggarwalpulkit596** [comment]
Can you assign a new issue for me ?

**Comment by rt4914** [comment]
> Can you assign a new issue for me ?

Please join this group: https://gitter.im/oppia/oppia-android

We normally have these type of conversations there. Also, by evening I will assign you new issues to work on.
